server/core/src/irods_api_test_harness.cpp

```diff
@@ -0,0 +1,17 @@
+#include "connection_pool.hpp"
+#include "experimental_plugin_framework.hpp"
+#include "irods_client_api_table.hpp"
+
+int main(int argc, char* argv[])
+{
+    if(argc < 3) { return 1; }
+
+    irods::api_entry_table&  api_tbl = irods::get_client_api_table();
+    irods::pack_entry_table& pk_tbl  = irods::get_pack_table();
+    init_api_table( api_tbl, pk_tbl );
```

Comment:
this can be replaced by a single call to `load_client_api_plugins()`.  it is available in `rodsClient.h`.